proto/internal_execution.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package internal_execution;
```

Comment:
This is fine, but you could also consider putting the new message in remote_execution.proto along with other BB specific-stuff. You could rename it slightly to make its meaning clear.